template.json

```diff
@@ -459,6 +459,12 @@
             "line": {
               "type": "long"
             },
+            "message": {
+              "doc_values": false,
+              "ignore_above": 1024,
```

Comment:
For logs, `ignore_above: 1024` will be too small.

Reply:
I checked the docs for this and run some tests: https://www.elastic.co/guide/en/elasticsearch/reference/current/ignore-above.html

It seems like `ignore_above` does not play any role here as it's not index anyway. `_source` is not affected by `ignore_above`.

Reply:
Good. Thanks for testing.

Maybe we shouldn't even set an `ignore_above` when `index: false`? I imagine others having a similar reaction to mine.